tests/baselines/reference/parser553699.errors.txt

```diff
@@ -1,4 +1,4 @@
-tests/cases/conformance/parser/ecmascript5/RegressionTests/parser553699.ts(3,21): error TS1216: Type expected. 'public' is a reserved word in strict mode. Class definitions are automatically in strict mode.
+tests/cases/conformance/parser/ecmascript5/RegressionTests/parser553699.ts(3,21): error TS1213: Identifier expected. 'public' is a reserved word in strict mode. Class definitions are automatically in strict mode.

```

Comment:
Or simply ditch the identifier bit and go with `'public' is a reserved word in strict mode. Class definition are automatically in strict mode.`